### ISSUE TITLE
[Question Only] Is it usable with InfluxDB 2.0

### ISSUE THREAD

**Issue by CatsPlugins** [influxdata/influxdb-php#139]
Hello,

 I'm using InfluxDB 1.7 with this PHP SDK, and it's working fine.

I want to upgrade to InfluxDB 2.0, so will it work with v2?

Thank you!

**Comment by andig** [comment]
Same question here?

**Comment by rawkode** [comment]
Yes, though work has begun on a client for 2.x. Please bare with us. I'll keep this issue updated over the coming days and weeks.

**Comment by rawkode** [comment]
I'm the interim, we suggest using 5
Telegraf as a proxy.

 https://www.influxdata.com/blog/influxdb-2-migration-path-instrumentation/

**Comment by andig** [comment]
> I'm the interim, we suggest using Telegraf as a proxy.

This is a little offtopic, please advise if there's a better place to ask. The link above suggests to use Telegraf for relaying. Thats good to know!

However, I already have `outputs.influxdb_v2` enabled for purpose of sending metrics to a `metrics` bucked. If I need to use it for `inputs.influxdb` -> `outputs.influxdb_v2` relay too, then it would need to go to a different bucket. How would I separate these two?

**Comment by rawkode** [comment]
You'd need to use the namepass/namedrop parameters on your 2 different InfluxDB outputs, or use a separate instance of Telegraf

**Comment by rawkode** [comment]
Example in this issue:

https://github.com/influxdata/telegraf/issues/809

**Comment by andig** [comment]
> Example in this issue: influxdata/telegraf#809

Thank you! Joined https://github.com/influxdata/telegraf/issues/1778 which seems to have a more detailed discussion.

**Comment by andig** [comment]
Unfortunately, as the docs say: 

> This plugin allows Telegraf to provide an InfluxDB 1 /write compatible proxy. This will not allow you to read from InfluxDB, should you be using any of the client libraries for that purpose.

As I'm periodically copying from another DB to influx (v1 at the moment) that won't work :/

**Comment by jadamroth** [comment]
Is there any update on InfluxDB 2.0 compatibility?

**Comment by rawkode** [comment]
@jadamroth No, not at the moment. With our recent beta announcement, this will change very soon :+1:

**Comment by jadamroth** [comment]
Hey @rawkode , any updates on timeline?

**Comment by bednar** [comment]
Hi @jadamroth,

we started working on [influxdb-client-php](https://github.com/influxdata/influxdb-client-php) for InfluxDB 2... first results coming soon

Regards
